src/app/progress-bar/progress-bar.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="progress-number">
+  {{ progress.current }} /
+  {{ progress.total }}
+</div>
+<div class="progress-bar">
```

Comment:
ProgressBar Component should just be the stuff in <div class='progress-bar'> the <div class='progress-number'> aren't really a progress bar.